api/app/serializers/v1/project_collection_serializer.rb

```diff
@@ -5,28 +5,35 @@ class ProjectCollectionSerializer < ManifoldSerializer
 
     abilities
 
-    typed_attribute :title, NilClass
-    typed_attribute :slug, NilClass
-    typed_attribute :pending_slug, NilClass
-    typed_attribute :sort_order, NilClass
-    typed_attribute :visible, NilClass
-    typed_attribute :homepage, NilClass
-    typed_attribute :position, NilClass
-    typed_attribute :icon, NilClass
-    typed_attribute :number_of_projects, NilClass
-    typed_attribute :featured_only, NilClass
-    typed_attribute :smart, NilClass
-    typed_attribute :description, NilClass
-    typed_attribute :description_formatted, NilClass
-    typed_attribute :description_plaintext, NilClass
-    typed_attribute :sort_order, NilClass
```

Comment:
This `sort_order` attribute was redundant in the original serializer and removed during updates.